.gitignore

```diff
@@ -1,2 +1,6 @@
-node_modules
-.env
\ No newline at end of file
+node_modules/
+src/generated/type-graphql
+src/generated/prisma-client
+src/generated-schema.graphql
+.env
+*.env
```

Comment:
Properly use dotenv package
Removed node_modules from repo
Ignores the env constants to the database url
Ignores generated schema from prisma